src/constants/chains.ts

```diff
@@ -63,65 +74,89 @@ export type ChainInfo = { readonly [chainId: number]: L1ChainInfo | L2ChainInfo
 
 export const CHAIN_INFO: ChainInfo = {
   [SupportedChainId.ARBITRUM_ONE]: {
+    blockWaitMsBeforeWarning: ms`10m`,
```

Comment:
Does this need to add XDAI?

Reply:
So the default value is `ms`10m`` but we can change this, what do you think it should be?